samples/CloudClipboard/CloudClipboard/appsettings.json

```diff
@@ -0,0 +1,6 @@
+{
+  "Logging": { "LogLevel": { "Default": "Warning" } },
+  "AllowedHosts": "*",
+  "BlobServiceUri": "https://cloudclips.blob.core.windows.net/",
+  "AZURE_INSTRUMENTATION_KEY": "Your App Insights Instrumentation Key"
```

Comment:
nit:  We should probably change the casing here to be consistent with other members.